daemon/CMakeLists.txt

```diff
@@ -571,6 +571,11 @@ if (USE_BREAKPAD)
         target_link_libraries(crash_server
             srclibs-breakpad-crash_generation_server srclibs-breakpad-common)
         target_compile_definitions(crash_server PRIVATE UNICODE)
+        if (MINGW)
```

Comment:
A better way of doing this is to add `WIN32` to `add_executable`: https://cmake.org/cmake/help/v3.0/command/add_executable.html